web/ui/react-app/src/utils/utils.test.ts

```diff
@@ -190,7 +190,7 @@ describe('Utils', () => {
       it('renders never for pre-beginning-of-time strings', () => {
         expect(formatRelative('0001-01-01T00:00:00Z', now())).toEqual('Never');
       });
-      it('renders a humanized duration for sane durations', () => {
+      it('renders a humanized duration for reasonable durations', () => {
```

Comment:
Not sure if "reasonable" is the right word here. Maybe "for correct duration strings"?